kube/minikube/deployments/tarball-deployment.yaml

```diff
@@ -0,0 +1,68 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
How are the credentials for the storage being passed to the container?

Reply:
I uploaded SSH key to the Minikube node.
Please see comments https://github.com/kernelci/kernelci-pipeline/pull/285#issuecomment-1670817784 and https://github.com/kernelci/kernelci-pipeline/pull/285#issuecomment-1671101131.

Reply:
OK, and then not defining any credentials in the TOML settings?  In principle it should have the SSH key name (when using SSH storage).

Reply:
Ah yes, sorry I missed that.  I was still thinking of the AKS deployment which has a secret token in `storage_cred` for Azure Files.